### ISSUE TITLE
`danger.github.pr` is undefined

### ISSUE THREAD

**Issue by flovilmart** [danger/peril#97]
# Background

I followed the steps to deploy on heroku for the full organization (parse-community) and got peril running with danger 1.0

# Configuration

The following env vars are set:
```
DATABASE_JSON_FILE = parse-community/peril@settings.json
PERIL_BOT_USER_ID = 26932219 // may be inaccurate
PERIL_INTEGRATION_ID = xxxx // working
PERIL_ORG_INSTALLATION_ID = xxxx
PERIL_WEBHOOK_SECRET = xxxx
PRIVATE_GITHUB_SIGNING_KEY = xxx
```

Reporting any message work as expected so peril is correctly configured from that standpoint.

# Logs

Now, when a PR event is received this is the log:

```
2017-07-01T16:06:15.262501+00:00 app[web.1]: info: [runner] - Event Settings: {
2017-07-01T16:06:15.262502+00:00 app[web.1]:  "commentableID": 10,
2017-07-01T16:06:15.262503+00:00 app[web.1]:  "hasRelatedCommentable": true,
2017-07-01T16:06:15.262503+00:00 app[web.1]:  "isRepoEvent": true,
2017-07-01T16:06:15.262503+00:00 app[web.1]:  "isTriggeredByUser": true,
2017-07-01T16:06:15.262504+00:00 app[web.1]:  "repo": null,
2017-07-01T16:06:15.262504+00:00 app[web.1]:  "repoName": "parse-community/peril",
2017-07-01T16:06:15.262505+00:00 app[web.1]:  "triggeredByUsername": "flovilmart"
2017-07-01T16:06:15.262505+00:00 app[web.1]: }
2017-07-01T16:06:15.310297+00:00 app[web.1]: info: [runner] - Running: {
2017-07-01T16:06:15.310300+00:00 app[web.1]:  "event": "pull_request",
2017-07-01T16:06:15.310300+00:00 app[web.1]:  "action": "synchronize",
2017-07-01T16:06:15.310301+00:00 app[web.1]:  "dslType": 0,
2017-07-01T16:06:15.310302+00:00 app[web.1]:  "dangerfilePath": "dangerfiles/pr.js",
2017-07-01T16:06:15.310302+00:00 app[web.1]:  "repoSlug": "parse-community/peril",
2017-07-01T16:06:15.310303+00:00 app[web.1]:  "feedback": 0
2017-07-01T16:06:15.310303+00:00 app[web.1]: }
2017-07-01T16:06:15.310410+00:00 app[web.1]: info: [runner] - Use fullDSL: true
2017-07-01T16:06:15.310539+00:00 app[web.1]: info: [runner] - supportGithubCommentAPIs: true
2017-07-01T16:06:15.362852+00:00 app[web.1]: info: [runner] - Running Danger: got API? null
2017-07-01T16:06:15.859265+00:00 heroku[router]: at=info method=POST path="/webhook" host=peril-parse-community.herokuapp.com request_id=11b79eec-7981-41ac-a851-9a212a47e2ce fwd="192.30.252.41" dyno=web.1 connect=1ms service=626ms status=200 bytes=216 protocol=https
2017-07-01T16:06:15.860777+00:00 heroku[router]: at=info method=POST path="/webhook" host=peril-parse-community.herokuapp.com request_id=46079c98-404f-4bca-954b-de3907f2fac8 fwd="192.30.252.42" dyno=web.1 connect=1ms service=165ms status=204 bytes=147 protocol=https
2017-07-01T16:06:15.852741+00:00 app[web.1]: info: [runner] - Commenting, with results:
2017-07-01T16:06:15.852763+00:00 app[web.1]: mds: 0
2017-07-01T16:06:15.852764+00:00 app[web.1]: mds: 0
2017-07-01T16:06:15.852765+00:00 app[web.1]: warns: 0
2017-07-01T16:06:15.852765+00:00 app[web.1]: fails: 1
2017-07-01T16:06:15.864118+00:00 app[web.1]: info: [router] -  - passing to Dangerfile rule router
2017-07-01T16:06:15.852766+00:00 app[web.1]:
2017-07-01T16:06:16.032212+00:00 app[web.1]: Request failed [403]: https://api.github.com/repos/parse-community/peril/statuses/f5a1c18e0a1f5e2afc66dad8d6c44ef02dcdcfc8
2017-07-01T16:06:16.032263+00:00 app[web.1]: Response: {
2017-07-01T16:06:16.032266+00:00 app[web.1]: }
2017-07-01T16:06:16.032264+00:00 app[web.1]:   "message": "Resource not accessible by integration",
2017-07-01T16:06:16.032265+00:00 app[web.1]:   "documentation_url": "https://developer.github.com/v3"
2017-07-01T16:06:16.032912+00:00 app[web.1]: Failing the build, there is 1 fail.
```

2 things that bug me:

1. github api is null ? `Running Danger: got API? null`
1. Resource not accessible, but all are configured as suggested with read access to commits etc...

**Comment by orta** [comment]
Interesting, let me write this PR and lets see if that fixes it :D

**Comment by flovilmart** [comment]
ahah! If you need me to deploy quickly I can test easily.

**Comment by orta** [comment]
Fixed by #99 - ash, there was also the repos issue
